rest-api/core-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v1/OperationService.java

```diff
@@ -106,27 +106,27 @@ public Object execute(final Operation operation) {
     }
 
     @Override
-    public CloseableIterable<Object> generateObjects(final GenerateObjects<Object> operation) {
+    public Iterable<Object> generateObjects(final GenerateObjects<Object> operation) {
```

Comment:
Why were these CloseableIterables before? Is there any type to replace this return type of a simple Closeable Iterable?

Reply:
Resolved (see comment below)